index.html

```diff
@@ -212,12 +233,12 @@
             
         </form>
 
-        <section> 
-        <p>
+        <div id="say-hello" class="hidden">
+        <h2>
             <span id="join">Say hello to</span>
             <b id="name">NAME</b>
             👋
```

Comment:
maybe this emoji should go at the start instead of the end, otherwise looks like part of the group or user name

Reply:
even if ppl think, it belongs to the name, this would not be really an issue if it is not at the end.

i think it  looks nice and friendly this way; it was the outcome of some discussions with rosano in the initial PR, i would not change this here - if it becomes an issue, we can think over